DisCatSharp.ApplicationCommands/ExtensionMethods.cs

```diff
@@ -69,10 +69,7 @@ public static ApplicationCommandsExtension GetApplicationCommands(this DiscordCl
 	public static async Task<IReadOnlyDictionary<int, ApplicationCommandsExtension>> GetApplicationCommandsAsync(this DiscordShardedClient client)
 	{
 		await client.InitializeShardsAsync().ConfigureAwait(false);
-		var modules = new Dictionary<int, ApplicationCommandsExtension>();
-		foreach (var shard in client.ShardClients.Values)
-			modules.Add(shard.ShardId, shard.GetExtension<ApplicationCommandsExtension>());
-		return modules;
+		return client.ShardClients.Values.ToDictionary(shard => shard.ShardId, shard => shard.GetExtension<ApplicationCommandsExtension>());
```

Comment:
is there maybe any other way besides using reflection utilities or are the performance impacts by it not significant enough?